libraries/RC_Channel/RC_Channel.h

```diff
@@ -252,6 +252,7 @@ class RC_Channel {
         VFWD_THR_OVERRIDE =  176, // force enabled VTOL forward throttle method
         MOUNT_LRF_ENABLE =   177,  // mount LRF enable/disable
         FLIGHTMODE_PAUSE =   178,  // e.g. pause movement towards waypoint
+        AUTOTUNE_SWITCH =    180, // auto tune tuning switch
```

Comment:
maybe this should be AUTOTUNE_TEST_GAINS?

Reply:
or maybe AUTOTUNE_TEST

Reply:
changed to "AUTOTUNE_TEST_GAINS"